organization/_posts/2019/2019-05-09-coordination.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "HSF Weekly Meeting #165, 9 May, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+*Present/Contributors*: Graeme Stewart, Tommaso Boccali, Marco Clemencic, Davide
+Costanzo, Serhan Mete, Andrea Valassi, Guilherme Amadio, Michel Jouvin, Heather
+Gray, Daniel Elvira, Ben Morgan, Agnieszka Dziurda
+
+## News, general matters
+  - European Strategy Update [<span class="underline">meeting in
```

Comment:
I would even use "symposium" rather than "meeting", as that's the right (big) scope :-).